### DISCUSSION TITLE
Should we release 0.68.3?

### DISCUSSION THREAD

**Discussion by fortmarek** [reactwg/react-native-releases#22] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/29

---

# Should we release 0.68.3?

Current Release: [0.68.2](https://github.com/facebook/react-native/releases/tag/v0.68.2)

Conversations on this thread are limited:
- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main
1.

**Comment by opayen** [comment]
We would like to have https://github.com/facebook/react-native/commit/114d31feeeb47f5a57419e5088c3cbe9340f757a cherry-picked to 0.68.
It is a fix to a regression that was introduced in 0.68 (see https://github.com/facebook/react-native/issues/33977)

**Comment by kelset** [comment]
a couple more commits got "requested" via https://github.com/facebook/react-native/pull/34326

**Comment by kelset** [comment]
one more thing we want for the next patch: once this is merged on main https://github.com/facebook/react-native/pull/34339 (and so it's validated that the changes are viable) a separate PR will be done against `0.68-stable` to introduce the same changes. We want to wait until that PR is up & merged before proceeding with 0.68.3

**Reply by kelset** [reply]
PR for 0.68: https://github.com/facebook/react-native/pull/34345
